hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -459,6 +459,46 @@ func (r *HostedClusterReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		}
 	}
 
+	// Reconcile etcd client MTLS secret if the control plane is using an unmanaged etcd cluster
+	if hcluster.Spec.Etcd.ManagementType == hyperv1.Unmanaged {
+		r.Log.Info("Reconciling etcd client mtls secret")
+		if hcluster.Spec.Etcd.Unmanaged == nil || len(hcluster.Spec.Etcd.Unmanaged.TLS.Client.Name) == 0 || len(hcluster.Spec.Etcd.Unmanaged.Endpoint) == 0 {
+			return ctrl.Result{}, fmt.Errorf("etcd metadata not specified for unmanaged deployment")
+		}
+		var src corev1.Secret
+		r.Log.Info("Retrieving etcd client mtls secret", "name", hcluster.Spec.Etcd.Unmanaged.TLS.Client.Name)
```

Comment:
I think this is probably unnecessary log noise

Reply:
removed